pages/mbs.html

```diff
@@ -0,0 +1,172 @@
+                MBS Lending chose the Mast Platform as their broker portal to deliver a cutting edge digital experience for both lenders and brokers, increase operational efficiencies, and help grow the MBS Lending brand.
+              </p>
+              <p>
+                MBS Lending is a subsidiary of The Melton Building Society that specialises in offering mortgages to borrowers with more complex credit histories. Underwriting is manual, intensive and applications were received via PDFs.
+              </p>
+
+              <h4>The problem</h4>
+              <p>
+                In April 2020 Dan Atkinson, Head of Intermediaries explains, “we were looking for an application origination portal for our credit repair lending, which due to its complexities around lending criteria, had been difficult to obtain via our existing system provider. <strong>We wanted something modern and user friendly, that would provide us with more control over lending rules and the ability to quickly and easily effect changes when necessary</strong>.”
+              </p>
+
+              <h4>The solution</h4>
+              <p>
+                MBS Lending chose the Mast Platform as their broker portal to deliver a cutting edge digital experience for both lenders and brokers, increase operational efficiencies, and help grow the MBS Lending brand.
+              </p>
+
+              <blockquote class="d-none d-lg-block bg-primary-alt" style="float: left; margin: -0.5rem 1.25rem 1.25rem -7rem; width: 60%">
+                <div class="h4 mb-2">
+                  &ldquo;The onboarding process was slick and efficient and Mast were incredibly observant and attentive to our needs.&rdquo;
+                </div>
+                <span class="text-small text-muted">Dan Atkinson, Head of Intermediaries, Melton Building Society</span>
+              </blockquote>
+
+              <p>
+                The Mast Platform stood out <strong>“as the system is capable of filtering out applications that would not meet our requirements in order to improve the quality, efficiency and conversion” of applications</strong>. Mast’s Policy and Product Engines allows MBS Lending to quickly react to the market and make changes to the platform without needing to write a single line of code.  And brokers agree, describing the platform as <strong>“smooth”, “easy to navigate”, and “ahead of its time”</strong>.
+              </p>
+
+              <h4>Setting up</h4>
+              <p>Mast was able to get MBS Lending up and running quickly. <strong>“The onboarding process was slick, efficient and Mast were incredibly observant and attentive to our needs and requirements.”</strong> Despite the specialist products, policies, and bespoke forms,  Mast had the MBS Lending pilot portal up and running within 12 weeks. This also included custom work on affordability calculations and user management. “It was a pleasure to work with a provider that will go the extra mile to say yes.”
```

Comment:
We have the 12 weeks onboarding as one of the ++ of the case study so I feel like we should `<strong>` it in the paragraph here as well